tests/template/elegant_import_data/deviance-rpn.lte.txt

```diff
@@ -1 +1 @@
-invalid rpn: "48e6 10 /x"
+invalid rpn="48e6 10 /x" error="unknown token: /x"
```

Comment:
I think having the error adds more context. But, maybe the old message was formatted the way it was for a reason?